handlers/handler-publickey-auth/src/main/java/org/apache/servicecomb/authentication/provider/RSAProviderTokenManager.java

```diff
@@ -16,13 +16,14 @@
  */
 package org.apache.servicecomb.authentication.provider;
 
-import com.google.common.cache.Cache;
-import com.google.common.cache.CacheBuilder;
+import static java.util.Objects.isNull;
```

Comment:
I'd prefer not to use static import, this will make code a bit more hard to review.